docs/getting_started.rst

```diff
@@ -1,96 +0,0 @@
-****************************
-Getting Started with Astropy
-****************************
-
-Importing Astropy
```

Comment:
This section  has some important context that probably should be somewhere in the docs. But I'm not sure where else it goes without an "overview"...?